TrailBlaze/View Controllers/RegisterViewController.m

```diff
@@ -50,6 +50,8 @@ - (void)registerUser {
     newUser.username = self.usernameField.text;
```

Comment:
@Rhomboic  some feedback on this commit in general: 
-  **We need to break up these commits much more** than what we're doing already. Taking this commit as an example, this should have been broken up into:  
    - Importing the new pods 
    - Updating navigationItem title
    - Creating the Run data structure
    - Creating the User data structure
    - Updating fetchMates method
    - Update RegisterViewController new user initialization
- There is still commented out code in both `Run.m` and `MatesViewController.m`. There should be **zero commented out code** in any commit we put up
- I recall you mentioning that Parse has realtime capabilities, is there a reason we're importing Firebase?